index.html

```diff
@@ -210,6 +214,18 @@
 
         const sortByLastName = (personList) => sortByFirstName(personList).reverse();
 
+        function filterPersonListByValidHeadshots(list){
+            return list.filter((person) => person.headshot.url != null );
```

Comment:
filter out all person objects whose headshots don't contain an image. This also fixes a bug when displaying headshots that don't contain an image.